pkg/proto/configuration/bb_runner/bb_runner.proto

```diff
@@ -76,4 +76,13 @@ message ApplicationConfiguration {
   // It is only safe to enable this option when this runner executes
   // only up to a single command concurrently.
   repeated string symlink_temporary_directories = 12;
+
+  // Run a command with arguments to clean the system for actions
+  // This commmand runs at least once before an action, and once immediately
+  // after an action.
+  //
+  // Caution; since this option executes a command in a separate process
+  // there is an inherent performance penalty. If possible use the built-in
+  // cleaning options that are handled in-process for optimal performance
```

Comment:
(Nit: Missing full stop at the end of the sentence.)

Thanks for adding this notice!